src/backend/db/database_manager.py

```diff
@@ -41,7 +41,7 @@ def _initialize_session_factory(self):
         logger.info(f"Connecting to database as user: {db_config['user']} on host: {db_config['host']}")
 
         engine = create_engine(
-            f"mysql+mysqlconnector://{db_config['user']}:{db_config['password']}@{db_config['host']}",
+            f"mysql+mysqlconnector://{db_config['user']}:{db_config['password']}@{db_config['host']}:{os.getenv('DB_PORT', '3306')}",
```

Comment:
Use it without the default value

Reply:
I removed the default values from this line